install.sh

```diff
@@ -4076,6 +4217,39 @@ install_hooks_into_repo() {
     fi
 }
 
+############################################################
+# Adds the repository to the list `autoupdate.registered`
+#  for potential future autoupdates.
+#
+# Returns: 0
+############################################################
+register_repo_for_autoupdate() {
+    CURRENT_REPO="$(cd "$1" && pwd)"
+    LIST="$INSTALL_DIR/autoupdate/registered"
+
+    # Remove
+    if [ -f "$LIST" ]; then
+        TEMP_FILE=$(mktemp)
+        CURRENT_ESCAPED=$(echo "$CURRENT_REPO" | sed "s@/@\\\/@g")
+        sed "/$CURRENT_ESCAPED/d" "$LIST" >"$TEMP_FILE"
```

Comment:
Couldn't we just do `sed "|$CURRENT_REPO|d"` and assume directories won't have the `|` character in their path?
I'm worried this double-sed might actually be more fragile.